test/parallel/test-https-req-split.js

```diff
@@ -33,8 +34,8 @@ const tls = require('tls');
 const fs = require('fs');
 
 const options = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fs.readFileSync(`${fixtures.fixturesDir}/keys/agent1-key.pem`),
+  cert: fs.readFileSync(`${fixtures.fixturesDir}/keys/agent1-cert.pem`)
```

Comment:
The `fs.readFileSync` calls could be replaced with `fixtures.readKey`.

Reply:
Thanks, test updated. Do I need to do anything else to update the PR?